rocAL/source/pipeline/master_graph.cpp

```diff
@@ -351,6 +378,10 @@ void MasterGraph::release() {
 
     if (_graph != nullptr)
         _graph->release();
+    for (auto& graph : _graphs) {
```

Comment:
Add a comment mentioning that _graph var is used for single loaders and _graphs var is used for multiple loaders since we are calling release for both of them but only one will be initialized at any time